server/handlers/clients.js

```diff
@@ -177,9 +171,9 @@ async function getConfigHandler(req, res) {
   }
 
   // scrub apiKeys and tokens
-  for (let key in configurations.data) {
+  for (let key in config.data) {
```

Comment:
Could we do something like:

```
config.data['api-key'] = undefined;
config.data['token'] = undefined;
```

no matter what, instead of iterating through them?
it would miss slight variations of `api-key` and `token` that this is catching

Reply:
Yeah, that's my concern. `wallet-api-key` would get through for example so I kind of feel like it's safer to just have the rule that nothing with `api` or `token` get through. Seems better to be on the safe side for something like this.

Reply:
Ok yeah that makes sense